packages/boilerplate/theme/nuxt.config.js

```diff
@@ -10,20 +10,20 @@ export default {
     title: 'Vue Storefront',
     meta: [
       { charset: 'utf-8' },
-      { name: 'viewport',
-        content: 'width=device-width, initial-scale=1' },
-      { hid: 'description',
-        name: 'description',
-        content: process.env.npm_package_description || '' }
+      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
+      { hid: 'description', name: 'description', content: process.env.npm_package_description || '' }
     ],
     link: [
       { rel: 'icon',
         type: 'image/x-icon',
         href: '/favicon.ico'
-      }, {
-        rel: 'preconnect',
-        href: 'https://fonts.googleapis.com'
-      }, {
+      },
+      {
+        rel: 'preload',
+        href: 'https://fonts.googleapis.com/css?family=Raleway:300,400,400i,500,600,700|Roboto:300,300i,400,400i,500,700&display=swap',
```

Comment:
As you are already using &display=swap, there should be no visual difference with this: https://css-tricks.com/the-fastest-google-fonts/
But Pagespeed should give better results

Reply:
if this is preloaded then we don't need a preconnect below